codecarbon/emissions_tracker.py

```diff
@@ -55,12 +55,18 @@ def __init__(
         self._output_dir: str = output_dir
         self._total_energy: Energy = Energy.from_energy(kwh=0)
         self._scheduler = BackgroundScheduler()
-        self._is_gpu_available = is_gpu_details_available()
-        self._hardware = GPU.from_utils(
-            gpu_ids
-        )  # TODO: Change once CPU support is available
+        self._hardware = []
+
+        if gpu.is_gpu_details_available():
+            logger.debug("Tracking Nvidia GPU")
+            self._hardware.append(GPU.from_utils(gpu_ids))
+        elif cpu.is_powergadget_available():
```

Comment:
Shouldn't this be a new `if` instead of an `elif` so that we can track both GPU and CPU?

Reply:
good catch !

Reply:
done